gnomad/sample_qc/sex.py

```diff
@@ -83,43 +178,62 @@ def get_ploidy_cutoffs(
     Uses the normal_ploidy_cutoff parameter to determine the ploidy cutoffs for XX and XY karyotypes.
     Uses the aneuploidy_cutoff parameter to determine the cutoffs for sex aneuploidies.
 
-    Note that f-stat is used only to split the samples into roughly 'XX' and 'XY' categories and is not used in the final karyotype annotation.
+    .. note::
+
+        `f_stat_cutoff` or `group_by_expr` must be supplied. If `f_stat_cutoff` is supplied then f-stat is used to
+        split the samples into roughly 'XX' and 'XY'. If `group_by_expr` is supplied instead, then it must include an
+        annotation grouping samples by 'XX' and 'XY'. These are both only used to divide samples into XX and XY to
+        determine means and standard deviations for these categories and are not used in the final karyotype annotation.
 
     :param ht: Table with f_stat and sex chromosome ploidies
-    :param f_stat_cutoff: f-stat to roughly divide 'XX' from 'XY' samples. Assumes XX samples are below cutoff and XY are above cutoff.
-    :param normal_ploidy_cutoff: Number of standard deviations to use when determining sex chromosome ploidy cutoffs for XX, XY karyotypes.
+    :param f_stat_cutoff: f-stat to roughly divide 'XX' from 'XY' samples. Assumes XX samples are below cutoff and XY
+        are above cutoff.
+    :param normal_ploidy_cutoff: Number of standard deviations to use when determining sex chromosome ploidy cutoffs
```

Comment:
I know its old but adding sd somewhere in this param would help with readability when it is called. Same goes for L193.

Reply:
hmm, I'm not sure the best option here. Is is worth the change when it will break code that assigns this parameter?

Reply:
No, for now, I don't think so